spec/features/projects/global_menu_item_spec.rb

```diff
@@ -32,10 +32,9 @@
 require "spec_helper"
 
 RSpec.describe "Projects global menu item", :js, :with_cuprite do
-  let(:user) { create(:user) }
+  current_user { create(:user) }
```

Comment:
In this case `shared_current_user` won't work because it is changed later (well, maybe it would work. I did not try). What I generally do in this case is this:
```
shared_let(:user) { create(:user) }
shared_let(:admin) { create(:admin) }
current_user { user }

context "with an admin user" do
  current_user { admin }
end
```